sentinelhub/time_utils.py

```diff
@@ -63,8 +69,9 @@ def iso_to_datetime(date):
     :return: datetime instance
     :rtype: datetime
     """
+    warnings.warn('This function will be removed in the next version', category=SHDeprecationWarning)
```

Comment:
I like the idea, but I think this might trigger quite a few changes which would get a bit out of the scope of this PR. So I'll open an internal issue about this and put it to "Wishlist".